src/edu/csus/ecs/pc2/clics/API202306/LanguageService.java

```diff
@@ -0,0 +1,118 @@
+// Copyright (C) 1989-2024 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+package edu.csus.ecs.pc2.clics.API202306;
+
+import java.util.ArrayList;
+
+import javax.inject.Singleton;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.Feature;
+import javax.ws.rs.core.FeatureContext;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.SecurityContext;
+import javax.ws.rs.core.Response.Status;
+import javax.ws.rs.ext.Provider;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import edu.csus.ecs.pc2.core.IInternalController;
+import edu.csus.ecs.pc2.core.model.Language;
+import edu.csus.ecs.pc2.services.core.JSONUtilities;
+import edu.csus.ecs.pc2.core.model.IInternalContest;
+
+/**
+ * WebService to handle languages
+ * 
+ * @author John Buck
+ *
+ */
+@Path("/contests/{contestId}/languages")
+@Produces(MediaType.APPLICATION_JSON)
+@Provider
+@Singleton
+public class LanguageService implements Feature {
+
+    @SuppressWarnings("unused")
```

Comment:
It's not clear why `model` is marked `unused`; it IS used in both `@GET` methods.

Reply:
Left over from original port of code to new API.  I removed it in the next push.